kubernetes/core-pod.yml

```diff
@@ -1,30 +1,30 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: busybox-pod
+  name: core-pod
   labels:
-    app: busybox
+    app: core-app
 spec:
   containers:
-  - name: busybox-container
-    image: busybox
+  - name: core-app-container
+    image: jmatejka/happy-bank-core:1.0.0
     args:
     - /bin/sh
     - -c
-    - touch tmp/healthy; sleep infinity;
+    - touch healthy; ls; sleep infinity;
     ports:
      - containerPort: 80
     readinessProbe:
       exec:
         command:
         - cat
-        - tmp/healthy
+        - healthy
       initialDelaySeconds: 15
       periodSeconds: 10
     livenessProbe:
       exec:
         command:
         - cat
-        - tmp/healthy
+        - healthy
       initialDelaySeconds: 20
       periodSeconds: 10
```

Comment:
Health checks could use http and REST api endpoint.

Reply:
Hi @JanMate. I've already added http health check with /health endpoint on my local branch. I just haven't pushed it yet. But thank you for the hint